app/build.gradle

```diff
@@ -118,8 +121,8 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
     implementation "com.squareup.okhttp3:okhttp:4.9.0"
 
-    //coil
-    implementation("io.coil-kt:coil:2.1.0")
+    implementation 'com.google.maps.android:android-maps-utils:2.4.0'
```

Comment:
add a comment to this library like the other ones

Reply:
Added comment